tests/etl/test_tasks.py

```diff
@@ -123,6 +123,159 @@ async def test_batch_is_given_schema(self):
         self.assertIn("id", schema.names)
 
 
+@ddt.ddt
+class TestTaskCompletion(TaskTestCase):
+    """Tests for etl__completion* handling"""
+
+    async def test_encounter_completion(self):
+        """Verify that we write out completion data correctly"""
+        self.make_json("Encounter.1", "A")
+        self.make_json("Encounter.2", "B")
+        self.make_json("Encounter.3", "C")
```

Comment:
nit: this may just be me, but generally i've been finding it hard to keep track of the context in my head when tests have IDs/Groups as a single character what the string is actually supposed to represent. Not saying change it, just something to consider.

Reply:
That's totes fair... I was using single characters for width savings, but it might not be worth the lower readability. I'll make a little pass and try to add a few more descriptive names.

Reply:
I renamed a few variables - and will try to keep an eye on myself in the future :smile: